diff --git a/meta-rcar-gateway/recipes-kernel/linux/linux-renesas_5.10.bb b/meta-rcar-gateway/recipes-kernel/linux/linux-renesas_5.10.bb
index 44a3ea64..e03bb19b 100644
--- a/meta-rcar-gateway/recipes-kernel/linux/linux-renesas_5.10.bb
+++ b/meta-rcar-gateway/recipes-kernel/linux/linux-renesas_5.10.bb
@@ -16,7 +16,6 @@ SRC_URI = "${RENESAS_BSP_URL};nocheckout=1;branch=${BRANCH};protocol=https \
     file://0002-arm64-dts-renesas-r8a779f0-Enable-IPMMU-main-and-HC-.patch \
     file://0003-arm64-dts-renesas-r8a779f0-Enable-IPMMU-for-PCIe0-1.patch \
     file://0004-arm64-dts-renesas-r8a779f0-Enable-IPMMU-for-eMMC.patch \
-    file://init_disassemble_info-signature-changes-causes-compile-failures.patch \
     file://ufs.cfg \
     file://r8a779f0_ufs.bin \
 "
